src/main/java/com/booster/cliclient/command/handler/DeleteNoteCommandHandler.java

```diff
@@ -0,0 +1,51 @@
+package com.booster.cliclient.command.handler;
+
+import com.booster.cliclient.command.Command;
+import com.booster.cliclient.console.OutputWriter;
+import com.booster.cliclient.console.UserInputReader;
+import lombok.RequiredArgsConstructor;
+import lombok.SneakyThrows;
+import okhttp3.OkHttpClient;
+import okhttp3.Request;
+import okhttp3.Response;
+import org.springframework.stereotype.Component;
+
+@Component
+@RequiredArgsConstructor
+public class DeleteNoteCommandHandler implements CommandHandler {
+
+    private final UserInputReader adapter;
+    private final OkHttpClient okHttpClient;
+    private final OutputWriter outputWriter;
+
+    @Override
+    @SneakyThrows
+    public void handle() {
+        outputWriter.print("Id");
+        String id = adapter.readLine();
+
+        Command cmd = Command.from(id);
+        if (cmd == Command.EXIT) {
+            return;
+        }
+
+        Request request = new Request.Builder()
+                .url("http://localhost:8081/note/" + id)
+                .delete()
+                .build();
+
+        try (Response response = okHttpClient.newCall(request).execute()) {
+            if (!response.isSuccessful()) {
+                outputWriter.print("Error occurred");
```

Comment:
In the previous PR, you extended a message with a response code. Do we need the same here?

Reply:
And you should use println method instead of print

Reply:
We use `print` in all handlers.

I am not sure about the `response code` for now. We will refactor this part and get rid of the duplication soon